src/main/webapp/script.js

```diff
@@ -30,62 +30,100 @@ async function getYouTubeComments() {
 }
 
 /**
- * Fetches data and adds to html
+ *  Fetches data and adds to html.
  */
 async function getChart() {
   $('form').submit(async function() {
     document.getElementById('loading-img').style.display = 'block';
     commentStats = await getYouTubeComments();
-    averageScore = commentStats.averageScore;
-    averageMagnitude = commentStats.averageMagnitude;
     sentimentBucketList = commentStats.sentimentBucketList;
+    wordFrequencyMap = commentStats.wordFrequencyMap;
 
-    const CommentSentimentTable = new google.visualization.DataTable();
-    CommentSentimentTable.addColumn('number', 'InclusiveStart');
-    CommentSentimentTable.addColumn('string', 'SentimentRange');
-    CommentSentimentTable.addColumn('number', 'CommentCount');
+    getBarChart(sentimentBucketList);
+    getWordCloudChart(wordFrequencyMap);
+    console.log(wordFrequencyMap);
+    averageScore = commentStats.averageScore;
+    const averageContainer = document.getElementById('average-score-container');
+    averageContainer.innerHTML = 'Average Sentiment Score: ' + averageScore;
+  });
+}
 
-    for (i = 0; i < sentimentBucketList.length; i++) {
-      currentSentimentBucket = sentimentBucketList[i];
-      rangeAsString = convertRangeToString(
-          currentSentimentBucket.intervalRange);
-      highestMagnitudeComments = currentSentimentBucket.topNComments;
+/**
+ * Create a bar chart of sentiment score interval, frequency
+ * and high magnitude comments
+ * @param {Array<sentimentBucket>} sentimentBucketList
+ */
+function getBarChart(sentimentBucketList) {
+  const CommentSentimentTable = new google.visualization.DataTable();
+  CommentSentimentTable.addColumn('number', 'InclusiveStart');
+  CommentSentimentTable.addColumn('string', 'SentimentRange');
+  CommentSentimentTable.addColumn('number', 'CommentCount');
 
```

Comment:
It looks like some of the tooltip logic around displaying top comments may have been removed during merging?

(Note that this didn't happen in this PR)

Reply:
I convert the tooltip logic to line 61 - 68, I can change it back and put it in another PR if needed